src/Triggers.s.sol

```diff
@@ -80,14 +85,40 @@ contract HookScript is Script {
         for (uint256 i = 0; i < target.length; i++) {
             string memory objectKey = "generic_target";
             vm.serializeString(objectKey, "trigger_type", target[i].triggerType.toString());
-            vm.serializeUint(objectKey, "chain_id", target[i].chainId.chainId);
+            vm.serializeUint(objectKey, "chain_id", target[i].chainId);
             vm.serializeString(objectKey, "target_type", "global");
             vm.serializeBytes32(objectKey, "listener_codehash", target[i].listenerCodehash);
+            BlockRange memory blockRange = target[i].blockRange;
+            serializeBlockRange(objectKey, blockRange);
+
             serializedTargets[i] = vm.serializeBytes32(objectKey, "handler_selector", target[i].handlerSelector);
         }
         return serializedTargets;
     }
 
+    function serializeBlockRange(string memory objectKey, BlockRange memory range) internal {
+        string memory rangeKey = string.concat(objectKey, "_range");
+        
+        if (range.kind == BlockRangeKind.FULL_SYNC) {
```

Comment:
We can leave these assertions for the CLI. It will be easier to provide helpful error messages there.

Reply:
Fixed in cdc459d987ebe3dbcf6ccfda5a232806fa963fc9